cert/CACreationScript.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+#=============
+#Copyright 2016 SAS Project Authors. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#=============
+
+echo Run this script from the directory containing the CBRS openssl.cnf file present in it. The openssl command must be present on the path.
+
+DIR=./root
+PAPA=$DIR/ca
+CERTI=$PAPA/certs
```

Comment:
I'm planning on updating some of these vars (that I set and completely forgot about) to be more descriptive/accurate.